templates/docs/examples/patterns/lists/lists-dividers.html

```diff
@@ -5,8 +5,14 @@
 
 {% block content %}
 <ul class="p-list--divided">
-  <li class="p-list__item">Single-command install and upgrade</li>
-  <li class="p-list__item">Postgres database for reliable state</li>
-  <li class="p-list__item">High availability for all services</li>
+  <li class="p-list__item">Balance compute load in the cloud</li>
+  <li class="p-list__item nested-list-wrapper">
+    <ul class="p-list--divided">
```

Comment:
Why it's nested in separate list element?

Nested lists are within the same list element. You have a list element and it has a child element list with more list elements.

```
  <li class="p-list__item">
    Balance compute load in the cloud
    <ul class="p-list--divided">
        <li> …
     </ul>
  </li>
```

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul

Reply:
:facepalm: 

Well... I did wonder why I didn't have to do that the first time around. It should be ok now